contrib/cni/README.md

```diff
@@ -0,0 +1,10 @@
+## `contrib/cni` ##
```

Comment:
Just thinking keep the style same with other places. But it's ok if you insist on this.